.github/workflows/build_wasm.yml

```diff
@@ -0,0 +1,66 @@
+name: build_wasm
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+
+jobs:
+
+  build_linux:
+    name: Build_Linux
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: Linux_EMCC_2
+          os: ubuntu-20.04
```

Comment:
Small question: Is there any particular reason for using this distribution + compiler version ?
Or conversely, will we need to build for different distributions + compiler versions.

Reply:
Not really. I just included the version I have in machine. Contrary to gcc, I think emscripten is not such a popular compiler so that we need to test with different versions

One open questions is wether we add a build for mac and windows. What do you think?

Reply:
A few points for prioritization:
1. External users of this module can set up their own custom build parameters. 
2. We do want a "minimum" test matrix which should cover all platforms. 
3. We don't need to decide now but at some point we'll need to set the exact configuration known to be supported for each platform. This can be decided at time of merge back into ILM main.
4. As we are also a "user" we can customize the default test matrix.

For this PR, let's get this in as a start, but would like to get Windows up and running as we have a number of folks working on that platform and would like anyone checking in changes to validate / fix issues on that platform. macos I think we can defer for a while.

Reply:
macos is exactly the same as for windows. Just a few things. But yes, for windows I tried for a couple o hours with no success. I'll create a follow up task. Mostly focusing in a explanation how to build it on windows including all emscripten configuration steps